### ISSUE TITLE
Incorrect Submission Status for Emissions in Evaluate screen

### ISSUE THREAD

**Issue by esaber76** [US-EPA-CAMD/PI-1#6068]
An incorrect Submission Status is displayed in the Evaluate screen for Emissions for quarters with closed windows.

Import a 2023 Q2 EM file from historical and navigate to the Evaluate screen.

Submission Status indicates "Permission to Resubmit Granted" and Submission Window indicates "Closed". 

**Requirements Note**:
In ECMPS 1.0 the Submission Availability Code for emissions was added to the EMISSION_EVALUATION table and used in view VW_EM_EVAL_AND_SUBMIT to determine the submission availability.  However, for emissions the submission availability is controlled by the EM_SUBMISSION_ACCESS table and the submission availability that it contains.  The following is SQL that shows how to pull the submission availability from the potentially multiple EM_SUBMISSION_ACCESS entries for each emission report.

[VW_EM_EVAL_AND_SUBMIT.ReplacementSql.txt](https://github.com/US-EPA-CAMD/easey-ui/files/14411902/VW_EM_EVAL_AND_SUBMIT.ReplacementSql.txt)

**Comment by annalbrecht** [comment]
Check EM_SUBMISSION_ACCESS table.

**Comment by djw4erg** [comment]
@maxdiebold-erg, you will need to manually update both views in the development database.  Definitely let me know if you need help with that.  I have meetings at 1 and 2, but should be relatively available before and after those meetings, although I usually end Mondays between 4 and 5pm EST.

**Comment by esaber76** [comment]
@maxdiebold-erg @djw4erg - did this change stop users from being able to evaluate a file if there is not a submission window? I created a 2023 Q4 file and cannot evaluate it. Nothing shows up for that quarter in the emissions section. I think that is a problem. Users often evaluate files prior to any windows getting created so they are prepared for the upcoming reporting period. This particular issue is also not a problem on tst.

**Comment by maxdiebold-erg** [comment]
@esaber76 Good catch. I think I've found the fix for the view, but we may need to look into the UI or an API, too, because I believe an empty window from this view has been interpreted as closed.

**Comment by djw4erg** [comment]
**Requirement Update**

@maxdiebold-erg, with the change to account for emissions with no windows, the frontend will need to account for a null window status, which should result in the window status "No Submission Window".

**Comment by maxdiebold-erg** [comment]
After further discussion with @djw4erg and @esaber76, the **Submission Window** and **Submission Status** columns should both show "None" in the case of a null window status.

**Comment by ntknguyen** [comment]
Trying to test this issue by imported 2023 Q2 emissions of Glenwood CP001, U00020, U00021, running the evaluation on emissions 2023 Q2 and encountered an endless loop.  Steps for testing:  

1. Evaluated the 2023 Q2 emissions.  
    The Eval Status shows In Progress, Submission Status shows 'Updated on Host', Submission Window shows 'Closed' 
     The evaluation process keeps running.   
2.  Updated the status_cd from 'WIP' to 'COMPLETE' in table EVALUATION_QUEUE
3.  'Revert to Official Record' stopped the endless loop.

@esaber76 - How do I test this issue without getting this endless loop?

**Comment by ntknguyen** [comment]
I re-tested and verified the issue is fixed.
